krkn/scenario_plugins/node_actions/abstract_node_scenarios.py

```diff
@@ -36,6 +36,16 @@ def helper_node_stop_start_scenario(self, instance_kill_count, node, timeout):
         self.helper_node_start_scenario(instance_kill_count, node, timeout)
         logging.info("helper_node_stop_start_scenario has been successfully injected!")
 
+    # Node scenario to detach and attach the disk
+    def disk_detach_attach_scenario(self, instance_kill_count, node, timeout, duration):
+        logging.info("Starting node_stop_start_scenario injection")
```

Comment:
let's be sure the logging information here matches the action we are doing, should be something like "Starting to detach disk"